scripts/deploy-multus-network.sh

```diff
@@ -38,12 +38,12 @@ if $TNF_NON_OCP_CLUSTER; then
 	kubectl -n kube-system wait --for=condition=ready -l name="cni-plugins" pod --timeout="$TNF_DEPLOYMENT_TIMEOUT"
 
 	# Install whereabouts at specific released version
-	git clone $WHEREABOUTS_GIT_URL --depth 1 -b v0.6.2
+	git clone $WHEREABOUTS_GIT_URL --depth 1 -b v0.6.3
 
 	# git clone $WHEREABOUTS_GIT_URL --depth 1
-	pushd whereabouts || exit
-	git checkout d15b4d5456ee910a81bf15e9e242a136823d22fd
-	popd || exit
+	# pushd whereabouts || exit
+	# git checkout d15b4d5456ee910a81bf15e9e242a136823d22fd
+	# popd || exit
```

Comment:
Was it a workaround? Avoid storing 'the history' in commented code; we have Git history for that.

Reply:
Yeah some sort of workaround.